### ISSUE TITLE
Add permissions logic for Plot Information edit

### ISSUE THREAD

**Issue by JakeWags** [visdesignlab/upset2#328]
When a user goes to edit the alt text `Plot Information` via the edit button, there should be a check to determine if they can edit this or not.

If they cannot edit, and they are logged in, they should not be allowed to edit at all. This could mean disabling the button or notifying the user on click.

If they are not logged in, ask the user to login.

**Comment by JakeWags** [comment]
Wait for https://github.com/visdesignlab/upset2/pull/363
